### ISSUE TITLE
Backpack Gui

### ISSUE THREAD

**Issue by RaoqTheWiz** [SpyMan10/backpackmod#3]
The Backpack Gui or when you open the packpack looks normal however when adding a resource pack that changes the inventory gui it makes it look wonky. My resource pack makes my inventory darker and the backpack mod when using bigger backpacks adds light theme to it and ruins it. Is there a way we can edit the gui?

**Comment by SpyMan10** [comment]
Yes I knew it because the gui is generated by using Minecraft chest gui texture, I will change that

**Comment by aisukuma** [comment]
Hello, I was wondering if you have managed to fix this issue?
